Minors (#1339)
hadley/r4ds: intro.qmd

## What you won't learn

> We believe it's important to stay ruthlessly focused on the essentials so you can get up and running as quickly as possible.

Reviewer: ```suggestion
There are several important topics that this book doesn't cover.
We believe it's important to stay ruthlessly focused on the essentials so you can get up and running as quickly as possible.
```

## What you won't learn

> There are several important topics that this book doesn't cover:
> we believe it's important to stay ruthlessly focused on the essentials so you can get up and running as quickly as possible.

Reviewer: ```suggestion
We believe it's important to stay ruthlessly focused on the essentials so you can get up and running as quickly as possible.
That means this book can't cover every important topic.
```
